src/components/questionStructure/Task/variants/HandwritingTask/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Pencil2Icon } from '@radix-ui/react-icons'
+import { Button, Card, Dialog, Flex, Text } from '@radix-ui/themes'
+import { MathJax } from 'better-react-mathjax'
+import { FC } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { DEFAULT_TEST_USERNAME } from '../../../../../utils/globalConstants'
+import { TaskType } from '../../constants'
+import { TaskBaseProps } from '../../types'
+import HandwritingEditor from './HandwritingEditor'
+import { ViewOnlyCanvas } from './ViewOnlyCanvas'
+import './index.scss'
+
+export interface HandwritingTaskProps extends TaskBaseProps<string> {
+  type: TaskType.PROCESSED_HANDWRITING
+  questionText: string
+}
+
+export const HandwritingTask: FC<HandwritingTaskProps> = ({
+  answer,
+  onAnswerUpdate,
+  disabled = false,
+  questionText,
+}) => {
+  const { username = DEFAULT_TEST_USERNAME } = useParams()
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <ViewOnlyCanvas initialData={{}} />
+      </Dialog.Trigger>
+      <Flex gap="3" align="center">
+        <Card className="latex-preview">
+          <Flex gap="3" p="3" align="center" justify="between">
+            <MathJax style={{ flex: '1 0 0', textAlign: 'center' }}>
+              <Text>{answer ? `\\( ${answer} \\)` : 'No Answer'}</Text>
+            </MathJax>
```

Comment:
It's this styling necessary? If it is, let's place it in a dedicated css file with a suitable class name. If not, let's get rid of it (I don't think the center alignment is actually needed here: all the order answers in the assessment would be left-justified by default, and that's ok).

Also, why `<Text>` around the answer? Perhaps not needed?

Reply:
Styling not needed so has been removed (Text isn't needed either)

Reply:
See d88c945